src/preset-themes/001.json

```diff
@@ -2,44 +2,46 @@
   "title": "001",
   "colors": {
     "toolbar": {
-      "h": 187,
-      "s": 77,
-      "l": 53,
-      "a": 100
+      "r": 43,
+      "g": 206,
+      "b": 227,
+      "a": 1
     },
     "toolbar_text": {
-      "h": 213,
-      "s": 43,
-      "l": 89
+      "r": 215,
+      "g": 226,
+      "b": 239
```

Comment:
Would it make sense to convert these to HEX at this point and just do something like `"toolbar_text": "#2e3848"`,
and `"toolbar": "#433d2d"`.

I couldn't actually find any color values with a non "0" or non "1" value. Which now actually seems pretty suspicious.

Reply:
> Would it make sense to convert these to HEX at this point

Not really - trying to keep these JSON files very close to / exactly matching the internal representation of redux app state to keep things simple. And so far, that means separate r/g/b/a properties